pkg/csi_driver/multishare_controller.go

```diff
@@ -57,9 +64,15 @@ type MultishareController struct {
 	isRegional                 bool
 	clustername                string
 	featureMaxSharePerInstance bool
+
 	// Filestore instance description overrides
 	descOverrideMaxSharesPerInstance string
 	descOverrideMinShareSizeBytes    string
+
+	pvLister       corelisters.PersistentVolumeLister
+	pvListerSynced cache.InformerSynced
+	kubeClient     *kubernetes.Clientset
```

Comment:
same here, kubeclient is already in `config.features.FeatureMaxSharesPerInstance`

Reply:
Fixed